view/theme/frio/templates/common_tabs.tpl

```diff
@@ -31,28 +33,33 @@
 			<ul class="tabs" role="menu">
 				{{foreach $tabs as $tab}}
 					{{if $tab.sel}}
-					<li id="{{$tab.id}}-xs" role="presentation" {{if $tab.sel}} class="{{$tab.sel}}" {{/if}}><a role="menuitem" href="{{$tab.url}}" {{if $tab.title}} title="{{$tab.title}}"{{/if}}>{{$tab.label}}</a></li>
+						<li id="{{$tab.id}}-xs" role="presentation" {{if $tab.sel}} class="{{$tab.sel}}" {{/if}}><a
+								role="menuitem" href="{{$tab.url}}" {{if $tab.title}} title="{{$tab.title}}"
+								{{/if}}>{{$tab.label}}</a></li>
 					{{else}}
-					{{$exttabs[]=$tab}}
+						{{$exttabs[]=$tab}}
 					{{/if}}
 				{{/foreach}}
 			</ul>
 		</li>
 
 		{{* All others are moved to this dropdown menu *}}
-		<li class="pull-right">
+		<li>
 			<ul class="tabs tabs-extended">
 				<li class="dropdown">
-					<button type="button" class="btn-link dropdown-toggle" id="dropdownMenuTools-xs" data-toggle="dropdown" aria-expanded="false">
+					<button type="button" class="btn-link dropdown-toggle" id="dropdownMenuTools-xs"
+						data-toggle="dropdown" aria-expanded="false">
 						<i class="fa fa-chevron-down" aria-hidden="true"></i>
 					</button>
-					<ul class="dropdown-menu pull-right" role="menu" aria-labelledby="dropdownMenuTools">
+					<ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenuTools">
 						{{foreach $exttabs as $tab}}
-						<li id="{{$tab.id}}-xs" role="presentation" {{if $tab.sel}} class="{{$tab.sel}}" {{/if}}><a role="menuitem" href="{{$tab.url}}" {{if $tab.title}} title="{{$tab.title}}"{{/if}}>{{$tab.label}}</a></li>
+							<li id="{{$tab.id}}-xs" role="presentation" {{if $tab.sel}} class="{{$tab.sel}}" {{/if}}><a
+									role="menuitem" href="{{$tab.url}}" {{if $tab.title}} title="{{$tab.title}}"
+									{{/if}}>{{$tab.label}}</a></li>
 						{{/foreach}}
 					</ul>
 				</li>
 			</ul>
 		</li>
 	</ul>
-</div>
+</div>
```

Comment:
Please configure your IDE to keep EOF EOL.

Reply:
For smarty tpl file my IDE (VSCode) is very limited as the only extension offer no configuration. Ill see what I can do.

Reply:
Ok found it, I usually don't have to play there with eslint/prettier autofixing everything for me :cat: